src/Raven.Client/Documents/Session/IAttachmentsSessionOperationsBaseOfTheBase.cs

```diff
@@ -0,0 +1,11 @@
+using Raven.Client.Documents.Operations.Attachments;
+
+namespace Raven.Client.Documents.Session;
+
+public interface IAttachmentsSessionOperationsBaseOfTheBase
```

Comment:
need to figure out the name, this one is not good

Reply:
we wont have different client api anymore so this will be removed